internal/reconcile/atomic_release.go

```diff
@@ -298,6 +310,11 @@ func (r *AtomicRelease) actionForState(ctx context.Context, req *Request, state
 		log.Info(msgWithReason("release not installed", state.Reason))
 
 		if req.Object.GetInstall().GetRemediation().RetriesExhausted(req.Object) {
+			if forceRequested {
+				log.Info(msgWithReason("forcing install while out of retries", "force requested through annotation"))
```

Comment:
Given that both annotations can be set at the same time, (force and reset), is this message accurate?

Reply:
Yes, as that means retries are no longer exhausted and this message will never be logged.

Reply:
Cool, ty/